libconnman-qt/vpnmodel.h

```diff
@@ -34,9 +34,9 @@
 #define VPNMODEL_H
 
 #include <QAbstractListModel>
+#include "vpnmanager.h";
 
 class VpnModelPrivate;
-class VpnManager;
```

Comment:
Don't remove the partial class that this intentional.

Reply:
Well this needs to be done for Qt6 by the looks of it due to static assertion. See https://stackoverflow.com/questions/66581395/q-property-must-be-fully-defined-error-in-qt-6 for more details.